Unit test CI build failure in pkg test

see https://ci.vmware.run/vmware/vic/4784
# github.com/vmware/vic/vendor/github.com/vmware/govmomi/object

vendor/github.com/vmware/govmomi/object/host_service_system.go:25: can't find import: "github.com/vmware/vic/vendor/github.com/vmware/govmomi/vim25/types"
Makefile:227: recipe for target 'test-job-github.com/vmware/vic/pkg' failed
make: **\* [test-job-github.com/vmware/vic/pkg] Error 2
make: **\* Waiting for unfinished jobs....

https://ci.vmware.run/vmware/vic/5528 is failing with this:
```
# github.com/vmware/vic/vendor/github.com/vmware/govmomi/vim25/mo
vendor/github.com/vmware/govmomi/vim25/mo/retrieve.go:23: can't find import: "github.com/vmware/vic/vendor/github.com/vmware/govmomi/vim25/methods"
Makefile:227: recipe for target 'test-job-github.com/vmware/vic/lib' failed
make: *** [test-job-github.com/vmware/vic/lib] Error 2
make: *** Waiting for unfinished jobs....
```

https://ci.vmware.run/vmware/vic/5591 (but now restarted)
```
# github.com/vmware/vic/vendor/github.com/vmware/govmomi/vim25/soap
vendor/github.com/vmware/govmomi/vim25/soap/client.go:37: can't find import: "github.com/vmware/vic/vendor/github.com/vmware/govmomi/vim25/types"
Makefile:227: recipe for target 'test-job-github.com/vmware/vic/lib' failed
make: *** [test-job-github.com/vmware/vic/lib] Error 2
make: *** Waiting for unfinished jobs....
```

--- a/Makefile
+++ b/Makefile
@@ -56,7 +56,7 @@ endif
 # Generate Go package dependency set, skipping if the only targets specified are clean and/or distclean
 # Caches dependencies to speed repeated calls
 define godeps
-  	$(call assert,$(call gmsl_compatible,1 1 7), Wrong GMSL version) \
+	$(call assert,$(call gmsl_compatible,1 1 7), Wrong GMSL version) \
 	$(if $(filter-out clean distclean .DEFAULT,$(MAKECMDGOALS)), \
 		$(if $(call defined,dep_cache,$(dir $1)),,$(info Generating dependency set for $(dir $1))) \
 		$(or \
@@ -234,7 +234,11 @@ sincemark:
 	@echo seconds passed since we start
 	@stat -c %Y /started | echo `expr $$(date +%s) - $$(cat)`
 
-test: portlayerapi $(TEST_JOBS)
+install-govmomi:
+# manually install govmomi so the huge types package doesn't break cover
+	$(GO) install ./vendor/github.com/vmware/govmomi
+
+test: install-govmomi portlayerapi $(TEST_JOBS)
 
 $(TEST_JOBS): test-job-%:
 	@echo Running unit tests

--- a/infra/scripts/coverage.sh
+++ b/infra/scripts/coverage.sh
@@ -70,8 +70,6 @@ show_cover_report() {
     go tool cover -${1}="$profile"
 }
 
-# manually install govmomi so the huge types package doesn't break cover
-go install ./vendor/github.com/vmware/govmomi
 generate_pkg_cover_data $(dir_to_pkg "$@")
 
 show_cover_report func

